src/pieces/assets/assets_command.py

```diff
@@ -38,26 +50,107 @@ class AssetsCommands:
 	current_asset = None
 
 	@classmethod
-	@check_assets_existence
-	@deprecated("open","list assets")
```

Comment:
think this might have been an accidental slip up , added it back in the commit [cdf18a8](https://github.com/pieces-app/cli-agent/pull/152/commits/cdf18a89688289d7141beb8b8637bc802c37375b)